etna/core/blocks/promoted_links.py

```diff
@@ -14,5 +14,5 @@ class PromotedLinkBlock(blocks.StructBlock):
 
 
 class AuthorPromotedLinkBlock(PromotedLinkBlock):
-    publication_date = blocks.DateBlock(required=False)
+    publication_date = blocks.CharBlock(required=False)
```

Comment:
@jamesbiggs What are we doing about existing pages / values here?

Reply:
I meant to ask you about these, I remember you had a PR that changed something from a `Char` to `RichText`, but couldn't find it anywhere to reference :(

How do I write a migration for this to move the data over?